config/routes.rb

```diff
@@ -51,6 +51,8 @@
   get '/admin', to: 'admin/dashboard#show', as: :admin_dashboard
   get '/admin/merchants/:id', to: 'admin/merchants#show', as: :admin_merchant_show
   get '/admin/merchants', to: 'admin/merchants#index', as: :admin_merchant_index
+  get '/admin/users', to: 'admin/users#index', as: :admin_user_index
+  get '/admin/users/:id', to: 'admin/users#show', as: :admin_user_show
```

Comment:
get '/admin/users/:id', to: 'admin/users#show', as: :admin_user_show

Pretty sure this should route to just 'users#show'

From User Story 52:

As an admin user
When I visit a user's profile page ("/admin/users/5")
I see the same information the user would see themselves
I do not see a link to edit their profile

Reply:
I'm a bit confused, I feel like the way I have it is right.  You think it should just link to a user's show page and not render the edit profile link?

Reply:
Yeah exactly, it says you should see the exact same show page just with one button missing. We might as well use the same controller/view action to keep the code DRY. Just throw in an if statement. That's in another user story though, so you don't have to worry about changing the user show view file

Reply:
This is an example of using scope without the module, we want the path for regular users and admins to route to the same method and then just control one button from there

Reply:
I was wrong! Doing it that way is DRY, but not secure. We need authorization. We can use a partial view if we want to DRY this up. PR looks good.